HeterogeneousCore/CUDAUtilities/test/cpu_unique_ptr_t.cpp

```diff
@@ -0,0 +1,31 @@
+#include "catch.hpp"
+
+#include "HeterogeneousCore/CUDAUtilities/interface/cpu_unique_ptr.h"
+
+TEST_CASE("cpu_unique_ptr", "[cudaMemTools]") {
+
+  SECTION("Single element") {
+    auto ptr = cudautils::make_cpu_unique<int>(0);
+    REQUIRE(ptr != nullptr);
+  }
+
+  SECTION("Reset") {
+    auto ptr = cudautils::make_cpu_unique<int>(0);
+    REQUIRE(ptr != nullptr);
+
+    ptr.reset();
+    REQUIRE(ptr.get() == nullptr);
+  }
+
+  SECTION("Multiple elements") {
+    auto ptr = cudautils::make_host_unique<int[]>(10,0);
```

Comment:
why this is `make_host_unique` instead of `make_cpu_unique` ?

Reply:
ops a bug